src/toil/batchSystems/gridengine.py

```diff
@@ -366,3 +366,10 @@ def setEnv(self, name, value=None):
             raise ValueError("GridEngine does not support commata in environment variable values")
         return AbstractBatchSystem.setEnv(self, name, value)
 
+    @staticmethod
+    def supportsWorkerCleanup():
+        """
+        Indicates whether the batch system supports the cleanup of workers on shutdown
```

Comment:
No need to duplicate the overridden method's docstring (unless you promise to keep these copies consistent from now until all eternity).